crates/holochain_conductor_api/src/config/conductor/dpki_config.rs

```diff
@@ -3,31 +3,28 @@
 
 use std::path::PathBuf;
 
+use cfg_if::cfg_if;
 use serde::Deserialize;
 use serde::Serialize;
 
-/// The network seed used in the main "production" DPKI network.
-const DPKI_NETWORK_SEED_MAIN: &str = "deepkey-main";
-
+#[cfg(feature = "unstable-dpki")]
 /// A network seed used for testing.
 const DPKI_NETWORK_SEED_TESTING: &str = "deepkey-testing";
 
 /// Configure which app instance ID to treat as the DPKI application handler
 /// as well as what parameters to pass it on its initialization.
```

Comment:
@jost it's outdated. If you look at the previous commit, this doc was created 5 years ago when the struct looked a lot different. Since you're leading the charge on this PR now perhaps you could update this.

Reply:
Alrighty